airflow/kubernetes/pod_launcher.py

```diff
@@ -31,8 +31,10 @@
 from kubernetes.stream import stream as kubernetes_stream
 from airflow import AirflowException
 from requests.exceptions import BaseHTTPError
+from urllib3 import HTTPResponse
 from .kube_client import get_kube_client
 
+POD_LOGS_POLL_INTERVAL_SECONDS = 5
```

Comment:
Where is this poll? Could this put pressure on the system if we have 20-100 active pods?

Reply:
This poll is the whole rest of the change lol. It could but I don't imagine it will be that much of an issue. we can bump it up to 10